dts/bindings/spi/st,stm32-spi-common.yaml

```diff
@@ -17,3 +17,7 @@ properties:
 
   pinctrl-names:
     required: true
+
+  ioswp:
+    type: boolean
+    description: Swap SPI MOSI and MISO pins
```

Comment:
there are some other vendor hardware which already have bindings doing similar things, at least I know for example nxp lpspi, maybe we should make a generic type of property

Reply:
Matching reference manual wordings is not as important as having a standard experience for zephyr users. That property on LPSPI is like that because the pins can be half duplex. I'm not saying to use that exact property how it is on LPSPI binding, that one could be reworked to fit however the generic scheme would be.